corehq/apps/sso/tests/test_oidc_utils.py

```diff
@@ -0,0 +1,17 @@
+from testil import eq
+
+from django.test import RequestFactory
+
+from corehq.apps.sso.utils.oidc import initialize_oidc_session
+
+
+def test_initialize_oidc_session():
+    request = RequestFactory().get('/sso/test')
+    request.session = {}
+    request.GET = {
+        'next': '/next/path',
+    }
+    initialize_oidc_session(request)
+    eq(request.session["oidc_state"] is not None, True)
+    eq(request.session["oidc_nonce"] is not None, True)
```

Comment:
I'm not sure that there's an elegant fix for this, but the assertion should be more than these values simply not being empty, right? If these values are sequential and easy to guess, for example, then this implementation isn't secure. I think we want to assert that these values are random. We could do so by monkey-patching rndstr(), but I'm not sure if that's worth it, as we make the test brittle, as now the test would break if the randomization function was changed, or if the order that these values were initialized was changed.

Reply:
yea, I don't think there is an elegant fix for this that wouldn't be brittle tbh 🤷